ROMFS/px4fmu_common/init.d-posix/airframes/4005_gz_x500_vision

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# @name Gazebo x500 vision
```

Comment:
I'd be tempted to just merge these things into a single full featured x500.

Reply:
But this means we would stop testing a barebones x500 no?

Reply:
We could control it PX4 side for different tests with and without VIO, GNSS, RNG, etc. 

I'm not sure if it's a good idea, just thinking out loud.